### DISCUSSION TITLE
How does this package relate to wink-sentiment, wink-post, etc.?

### DISCUSSION THREAD

**Discussion by tomhicks** [winkjs/wink-nlp#33] [General]
I've been using various wink packages for some time, and they're great!

I just realised that this package existed today, and now I'm wondering if I should have been using it instead of the individual packages. It doesn't seem this is listed on your website. Is it new?

Any advice on how you see the whole family of libraries relating to each other?

**Comment by sanjayaksaxena** [comment]
Hello @tomhicks 

We are happy to learn that you are finding winkjs useful.

Yes `wink-nlp` is new. We are in the process of releasing it — a new generation of NLP library. Currently the work on user documentation is in progress, the English language model is being refined, and showcases to illustrate use cases are being setup. All of this should get completed in few weeks.  

It includes tokenization, sentence boundary detection, negation handling, sentiment analysis, named entity detection, pos tagging and custom entities (user definable) definition & detection. Apart from a developer friendly API, there will be a number utility functions such as bag-of-word, ngram, frequency table, etc. 

Request you to see the [documentation (WIP)](https://winkjs.org/wink-nlp/). You may like to explore **showcase** repos (WIP) on **winkjs** homepage on Github.

We plan to retire the existing family of package within 12 months from the date we release the migration guidelines & documentation. This will take 2-3 months after the release of wink-nlp.

For you immediate requirements, you can continue to use the current family and it will be well supported. 

If you wish to use the early release of wink-nlp, please let us know… we will be delighted to support you. You can write to us at wink@graype.in also.

Best,
Sanjaya

**Comment by sanjayaksaxena** [comment]
Hello @tomhicks 

Hope this and the email answers  all your questions. Closing the issue!

Best,
Sanjaya
